src/requirements/data/index.ts

```diff
@@ -428,5 +437,6 @@ const json: RequirementsJson = {
 
 export const colleges = ['AG', 'AR', 'AS1', 'AS2', 'EN', 'HE', 'IL', 'BU'] as const;
 export type College = typeof colleges[number];
-
 export default json;
+
+export const getCollegeAbbr = (collegeCode: string) => json.college[collegeCode].abbr;
```

Comment:
Will this throw an exception if `collegeCode` is not a key of `json.college`? I think it'd be good practice to add in little redundancy checks, for example
```suggestion
export const getCollegeAbbr = (collegeCode: string) => json.college[collegeCode]?.abbr ?? 'other';
```